src/tools/fuzzing.h

```diff
@@ -323,28 +323,33 @@ class TranslateToFuzzReader {
     }
     SmallVector<Type, 2> options;
     options.push_back(type); // includes itself
-    TODO_SINGLE_COMPOUND(type);
-    switch (type.getBasic()) {
-      case Type::anyref:
-        if (wasm.features.hasReferenceTypes()) {
-          options.push_back(Type::funcref);
-          options.push_back(Type::externref);
-          if (wasm.features.hasExceptionHandling()) {
-            options.push_back(Type::exnref);
+    if (type.isFunction()) {
+      // TODO: specific typed function references types
+      options.push_back(Type::funcref);
```

Comment:
What's this change for? If `type` is a typed function reference, then this isn't correct. If `type` is a funcref, then this change isn't necessary.